<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/apps/Task1/icons/signal.png</normaloff>E:/apps/Task1/icons/signal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="4" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_3">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_2">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_7">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_8">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_9">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_10">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="8" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_6">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_4">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" alignment="Qt::AlignHCenter">
       <widget class="QSlider" name="verticalSlider_5">
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>60Hz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>170Hz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>310Hz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>600Hz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>1KHz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>3KHz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>6KHz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>12KHz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>14KHz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>16KHz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="PlotWidget" name="graphWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Palette 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Palette 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Palette 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Palette 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Palette 5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="PlotWidget" name="graphWidget2" native="true">
        <property name="minimumSize">
         <size>
          <width>650</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_signal"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionSave_signal_as"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuSelect_signal">
     <property name="title">
      <string>Select signal</string>
     </property>
     <addaction name="actionSignal1_2"/>
     <addaction name="actionSignal2_2"/>
     <addaction name="actionSignal3"/>
    </widget>
    <addaction name="ClearSignal"/>
    <addaction name="AddChannel"/>
    <addaction name="DeleteChannel"/>
    <addaction name="menuSelect_signal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="OpenSignalBtn"/>
   <addaction name="ZoomIn"/>
   <addaction name="ZoomOut"/>
   <addaction name="Beginning"/>
   <addaction name="PlayBtn"/>
   <addaction name="End"/>
   <addaction name="Pause"/>
   <addaction name="Stop"/>
   <addaction name="AddPanel"/>
   <addaction name="DeletePanel"/>
   <addaction name="spectrogram"/>
   <addaction name="PDF"/>
   <addaction name="LeftScroll"/>
   <addaction name="RightScroll"/>
  </widget>
  <action name="actionOpen_signal">
   <property name="text">
    <string>Open signal</string>
   </property>
  </action>
  <action name="actionSave_signal_as">
   <property name="text">
    <string>Save signal as </string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="OpenSignalBtn">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/open-folder-with-document.png</normaloff>E:/apps/Task1/icons/open-folder-with-document.png</iconset>
   </property>
   <property name="text">
    <string>Open Signal</string>
   </property>
   <property name="toolTip">
    <string>Open Signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="ZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/zoom-in.png</normaloff>E:/apps/Task1/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="ZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/zoom-out.png</normaloff>E:/apps/Task1/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="Beginning">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/backward.png</normaloff>E:/apps/Task1/icons/backward.png</iconset>
   </property>
   <property name="text">
    <string>Beggining</string>
   </property>
   <property name="toolTip">
    <string>Beggining</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="PlayBtn">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/play-button.png</normaloff>E:/apps/Task1/icons/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Play signal</string>
   </property>
   <property name="toolTip">
    <string>Play signal</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="End">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/forward.png</normaloff>E:/apps/Task1/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="toolTip">
    <string>End</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="Pause">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/pause.png</normaloff>E:/apps/Task1/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="spectrogram">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/Spectrogram.png</normaloff>E:/apps/Task1/icons/Spectrogram.png</iconset>
   </property>
   <property name="text">
    <string>Spectrogram</string>
   </property>
   <property name="toolTip">
    <string>Spectrogram</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="PDF">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/pdf.png</normaloff>E:/apps/Task1/icons/pdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="toolTip">
    <string>PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="ClearSignal">
   <property name="text">
    <string>Clear Signal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="AddChannel">
   <property name="text">
    <string>Add Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="DeleteChannel">
   <property name="text">
    <string>Delete Channel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="AddPanel">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/addChannel.png</normaloff>E:/apps/Task1/icons/addChannel.png</iconset>
   </property>
   <property name="text">
    <string>Add panel</string>
   </property>
   <property name="toolTip">
    <string>Add Panel</string>
   </property>
  </action>
  <action name="DeletePanel">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/delete.png</normaloff>E:/apps/Task1/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Panel</string>
   </property>
   <property name="toolTip">
    <string>Delete Panel</string>
   </property>
  </action>
  <action name="LeftScroll">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/left.png</normaloff>E:/apps/Task1/icons/left.png</iconset>
   </property>
   <property name="text">
    <string>Scroll Left</string>
   </property>
   <property name="toolTip">
    <string>Scroll Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="RightScroll">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/right.png</normaloff>E:/apps/Task1/icons/right.png</iconset>
   </property>
   <property name="text">
    <string>Right Scroll</string>
   </property>
   <property name="toolTip">
    <string>Right Scroll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionSignal2">
   <property name="text">
    <string>Signal2</string>
   </property>
  </action>
  <action name="actionSignal1_2">
   <property name="text">
    <string>Signal1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSignal2_2">
   <property name="text">
    <string>Signal2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionSignal3">
   <property name="text">
    <string>Signal3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="Stop">
   <property name="icon">
    <iconset>
     <normaloff>E:/apps/Task1/icons/stop.png</normaloff>E:/apps/Task1/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_6</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_5</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>722</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_7</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>792</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>810</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
